src/components/ColumnDefinition.js

```diff
@@ -24,7 +24,7 @@ export default class ColumnDefinition extends Component {
     customComponent: React.PropTypes.object,
 
     //The component that should be used instead of the normal title
-    customHeaderComponent: React.PropTypes.object,
+    customHeadingComponent: React.PropTypes.object,
```

Comment:
Yea, not necessary to this one but we should probably make this a `node` -- something that can be rendered instead of just an object.